userspace/libsinsp/test/plugins.ut.cpp

```diff
@@ -649,6 +646,13 @@ TEST_F(sinsp_with_test_input, plugin_syscall_async) {
 	auto ext_pl = register_plugin(&m_inspector, get_plugin_api_sample_syscall_extract);
 	add_plugin_filterchecks(&m_inspector, ext_pl, srcname, filterlist);
 
+	// Check that the `sampleticker` async events was added to the list of events
+	auto event_set =
```

Comment:
Test of the new feature.

Reply:
I added it here in the `plugin_syscall_async` test because we already register an async plugin here and have access to the inspector.